### DISCUSSION TITLE
no plugin button

### DISCUSSION THREAD

**Discussion by AnsenBU** [pear-devs/pear-desktop#711] [Q&A] [answered]
i no longer see the menu at the top? is there a keybind to get it back?

**Comment by Araxeus** [comment, marked as answer]
You probably activated the `hide menu` option

to show the menu, press<kbd>alt</kbd> 
or <kbd>escape</kbd> if using the `in-app-menu` plugin

**Comment by inson1** [comment]
@AnsenBU or @th-ch What about closing this discussion? :) its answered and duplicate of #358
